### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GnaphronG**
Update versions
Remove obsolete info
Add Licenses

### CODE REVIEW

**Comment by jrottenberg** [comment]
I think the table of size per image needs a refresh as well : 
` u-3.2               ubuntu              1e998987f2da        2 minutes ago        205.5 MB
 c-3.2               centos              b73e2e768092        9 minutes ago        274.4 MB
 a-3.2               alpine              5da8a4eaeb41        45 minutes ago       73.64 MB`

**Comment by jrottenberg** [comment]
fribidi ?! Failing on 

```
  curl -sLO http://fribidi.org/download/fribidi-${FRIBIDI_VERSION}.tar.bz2
```

File: README.md [hunk 1]
```diff
@@ -1,32 +1,35 @@
-FFMPEG for Docker on Centos7
-============================
+FFMPEG for Docker
```
Comment (jrottenberg) [inline README.md]: so FFmpeg ?

**Comment by GnaphronG** [comment]
But for docker

**Pull request merged**